apps/cli/src/store/local/selectors.ts

```diff
@@ -52,7 +53,6 @@ export function selectOxaLinkInformation(state: RootState, oxa: string) {
     title: fileInfo.title,
     description: fileInfo.description,
     url: info.url,
-    // TODO: thumbnail
-    thumbnail: undefined,
+    thumbnail: fileInfo.thumbnail || undefined,
```

Comment:
Ah, I guess it's ok, since this is just getting oxa link information. At this point, previously `undefined` values have already been attempted to resolve. So now for the oxa link, we say: "if the thumbnail was not defined and couldn't be resolved _or_ if it was always null, there is no thumbnail"